docs/spec.json

```diff
@@ -1523,4 +1522,4 @@
       "in": "query"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
The file must exactly match the file created by make update-swagger otherwise the build fails. The generated file does not contain a newline at the end.